doc/parameters/param_list.md

```diff
@@ -372,6 +372,7 @@ Namespace: /parent_ns/local_ns
 | ----------| --------| ------------|
 | planner_plugin_ids | ["GridBased"] | List of Mapped plugin names for parameters and processing requests |
 | planner_plugin_types | ["nav2_navfn_planner/NavfnPlanner"] | List of registered pluginlib planner types to load |
+| expected_planner_frequency | 20.0 | Expected planner frequency. If the current frequency is less than the expected frequency, display the warning message |
```

Comment:
Please add to our website too https://github.com/ros-planning/navigation.ros.org/blob/master/configuration/packages/configuring-planner-server.rst

Reply:
> Please add to our website too https://github.com/ros-planning/navigation.ros.org/blob/master/configuration/packages/configuring-planner-server.rst

I've created PR https://github.com/ros-planning/navigation.ros.org/pull/32